tests/TestTrust.py

```diff
@@ -226,3 +244,15 @@ def test_keyIOFails(self):
     def test_signNonexisting(self):
         private_key, public_key = TrustBasics.generateNewKeyPair()
         assert TrustBasics.getFileSignature("file-not-found", private_key) is None
+
+    def test_isPathInLocation(self):
```

Comment:
I think this test should be re-written with parametrize. Best way to do that would be to have two different tests; One that asserts truth, other that asserts false.

Although it's a bit more code to parametrize it, i personally prefer it, since it means that all the asserts are executed, even if an earlier one fails.

Reply:
Sure, I'll start on that when I'm done with these unit-tests.

Reply:
Fixed in https://github.com/Ultimaker/Uranium/pull/717/commits/81250f7af3c809cf213a6464016698bfeb9c0c12